product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/index.mdx

```diff
@@ -30,15 +31,20 @@ You can modify your cluster by modifying:
     | ----------------------------------- | ----------------------------------------------------------------------------------------- |
     | Cluster name and password           | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab)                 |
     | Instance type \*                     | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab) |
-    | Networking type (public or private) \**| [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab)|
-    | Database configuration parameters   | [DB Configuration](05_db_configuration_parameters)                       |
-    
+    | Volume properties \**               |  [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab) |
+    | Networking type (public or private) \*** | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab)|
+    | Database configuration parameters   | [DB Configuration](05_db_configuration_parameters)                       |    
 
-    \* Changing the instance type could incur higher cloud infrastructure charges.
+     
+     \* Changing the instance type could incur higher cloud infrastructure charges.
 
-    \** If you are using Azure and previously setup a private link and want to change to a public network, you must remove the private link resources before making the change.
+   \** It can take upto a few hours to tune and resize the disks in your cluster. The volume properties are disabled and cannot be modified while this is in progress.
```

Comment:
```suggestion
   \** If you are using AWS General Purpose SSD storage, it can take up to a few hours to tune and resize the disks in your cluster. The volume properties are disabled and cannot be modified while this is in progress.
```
I'm pretty sure this only applies to AWS GP3 storage so you need to state that.